### DISCUSSION TITLE
EL7 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#223] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/EL7.html

Author: @tdegeorge 
Reviewer: @AbbyANoble

**Comment by tdegeorge** [comment]
I switched things around a little from what we originally had in the outline....let me know what you think!

**Comment by AbbyANoble** [comment]
[Activity 5.7.2](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#exponential-application-1) - I'd suggest time elapsed instead of time. And degree**s** Celsius.

I'd also include the units in all the answer choices. `<m>20</m><degree/><m>C</m>`, etc.

The answer to (a) should be D, not B. 

A hint may be helpful in (c). Maybe "Think about what happens to the function as x -> infinity?"

**Comment by AbbyANoble** [comment]
[Activity 5.7.3(b)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#exponential-application-2-3) - This is probably exceptionally pedantic, but it would be the 18th day when the 2500th view happens. I get that rounding does make some sense, but I like the interpretation of realizing it must be the 18th day.

**Comment by AbbyANoble** [comment]
[Activity 5.7.3(b)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#exponential-application-2-3) - There's an extra space between the comma and the 5 in 2,500. But I think this is an issue in math mode? I've run into this and just used smaller numbers to avoid it.  😂 Maybe @siwelwerd or @StevenClontz can help with that formatting?

**Reply by AbbyANoble** [reply]
This happens in [Activity 5.7.9](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#compound-interest) too.

**Reply by AbbyANoble** [reply]
And in [Activity 5.7.12](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#continuous).

**Reply by StevenClontz** [reply]
Just tagging @StevenClontz again so I can be sure to have this in my inbox. (-:

**Comment by AbbyANoble** [comment]
[Activity 5.7.4](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#exponential-application-3) - I might break this into more steps. I read it thinking you wanted N(t) right away in part (a). It wasn't clear that we were writing this first equation so we could find the growth factor. Maybe something like:

In 2006, 80 deer were introduced into a wildlife refuge. By 2012, the population had grown to 180 deer. The population was growing exponentially. Recall that the general form of an exponential equation is f(x)=ab^x, where a is the initial value, b is the growth/decay factor, and t is time. We want to write a function N(t) to represent the deer population after t years.

a) What is the initial value for the deer population?
b) We are not given the growth factor, so we must solve for it. Write an exponential equation using the initial population, the 2012 population, and the time elapsed. 
c) Your part (b).
d) Now your (c).
e) (an application)... like if the growth continues according to this exponential function, when will the population reach 250?

**Comment by AbbyANoble** [comment]
[Activity 5.7.9(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#compound-interest-2) - I'd add "after t years" to the end of the question. 

Also, the 18 years in the questions feels like a distraction since you didn't come back to it.

**Comment by AbbyANoble** [comment]
[Remark 5.7.11](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#EL7-3-12) - r  is the continuous growth rate per **of** unit time in both spots where you define r. 

And you need a space in the last line after the comma and before and.

**Comment by AbbyANoble** [comment]
[Activity 5.7.12(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#continuous-2) - The word nominal hasn't been used or defined before here. I'd leave it out or explain it.

**Comment by AbbyANoble** [comment]
[Activity 5.7.12(b)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#continuous-3) - Include the units in the answer. 

I think it would be good to include units everywhere in this section.
